include/internal/refcount.h

```diff
@@ -174,19 +174,19 @@ static __inline int CRYPTO_GET_REF(CRYPTO_REF_COUNT *refcnt, int *ret)
 
 static __inline int CRYPTO_UP_REF(CRYPTO_REF_COUNT *refcnt, int *ret)
 {
-    *ret = _InterlockedExchangeAdd(&refcnt->val, 1) + 1;
+    *ret = _InterlockedExchangeAdd((void *) &refcnt->val, 1) + 1;
```

Comment:
A very minor style nit: we don't normally put a space after the cast:
```suggestion
    *ret = _InterlockedExchangeAdd((void *)&refcnt->val, 1) + 1;
```